.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,3 @@
+blank_issues_enabled: true
+[README.md](../README.md)
+[ESP32 forum](https://www.esp32.com/)
```

Comment:
I would suggest adding here structure with Description and link.

For example:

```
contact_links:
  - name: BSP Repository README
    url: https://github.com/espressif/esp-bsp/blob/master/README.md
```

Reply:
Updated